frontend/tests/dashboards/utils.ts

```diff
@@ -0,0 +1,180 @@
+import { Page } from '@playwright/test';
+import { JsonApplicationType } from '../fixtures/constant';
+
+// API endpoints
+export const getDashboardsListEndpoint = 'v1/dashboards';
```

Comment:
Please create `constants.ts` and shift all the constants there.

Reply:
will take care of this in follow-up PR for all three pages

Reply:
Works 👍